applications/DEMApplication/custom_elements/spheric_continuum_particle.h

```diff
@@ -155,7 +155,7 @@ namespace Kratos
                                                 const ProcessInfo& r_process_info,
                                                 array_1d<double, 3>& rElasticForce,
                                                 array_1d<double, 3>& rContactForce,
-                                                double& RollingResistance) override final;
```

Comment:
This `final `was added because this function is very big (our bad design) and, if overriden, many lines of code are duplicated. I proposed an alternative solution in a previous comment.

Reply:
Here I do not understand what can I change. The function ComputeBallToBallContactForce is called from SphericParticle::CalculateRightHandSide. Are you suggesting to override CalculateRightHandSide?

Reply:
Sorry, I did not remember that the `SphericParticle.cpp` and the `SphericContinuumParticle.cpp` were in such a different state of maintenance. The `SphericParticle` function called `ComputeBallToBallContactForce` was cleaned up by @GuillermoCasas , but nobody cleaned up the same function in `SphericContinuumParticle`.
Ok, the `final` must be removed. Too bad that the function is still so ugly!